torch/distributions/constraints.py

```diff
@@ -206,16 +214,19 @@ def support(self):
 
     def __init__(
         self,
-        fn: Optional[Callable[..., Any]] = None,
+        fn: Optional[Callable[[T], R]] = None,
```

Comment:
Why not ParamSpec here? T isn't used elsewhere?

Reply:
`ParamSpec` does not make sense here, since this is a property-decorator, and properties are usually not supposed to take arguments. (note that `T` is actually the type the property gets bound to)